crates/polars-time/src/chunkedarray/string/infer.rs

```diff
@@ -47,12 +47,12 @@ const DATETIME_YMD_PATTERN: &str = r#"(?x)
         (?:\d{1,2})                # day
         (?:
             [T\ ]                  # separator
-            (?:\d{2})              # hour
+            (?:\d{1,2})              # hour
```

Comment:
now the comments are no longer aligned, could you keep them aligned please?

Reply:
aligned